ql/pom.xml

```diff
@@ -1142,6 +1143,12 @@
                     <exclude>META-INF/licenses/slf4j*/**</exclude>
                   </excludes>
                 </filter>
+                <filter>
+                  <artifact>oorg.apache.tez:tez-protobuf-history-plugin</artifact>
```

Comment:
Shouldn't it be "org" instead of "oorg"?

Reply:
yeah, silly typo :) let me fix